src/pytestqt/qt_compat.py

```diff
@@ -110,7 +117,7 @@ def _import_module(module_name):
         self.qInstallMsgHandler = None
         self.qInstallMessageHandler = None
 
-        if self.pytest_qt_api == "pyside2":
+        if self.pytest_qt_api == "pyside2" or self.pytest_qt_api == "pyside6":
```

Comment:
Maybe also set a `self.is_pyside` when setting `self.pytest_qt_api`? Then this doesn't need to be repeated everywhere.

Reply:
changed in  f6397ee.